parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -609,31 +610,37 @@ notifyUser dir o = case o of
 
   TodoOutput names todo -> pure (todoOutput names todo)
   GitError input e -> pure $ case e of
+    CouldntParseRootBranch repo s -> P.wrap $ "I couldn't parse the string"
+      <> P.red (P.string s) <> "into a namespace hash, when opening the repository at"
+      <> P.group (prettyRepoBranch repo <> ".")
```

Comment:
Started using `prettyRepoRevision` and `prettyRepoBranch` throughout to consolidate the formatting.